integration/src/integration/java/pl/allegro/tech/hermes/integration/tracker/MongoMessageTrackingTest.java

```diff
@@ -234,11 +233,11 @@ public void shouldToggleTrackingOnSubscriptionUpdate() {
     public void shouldReturnEmptyListIfThereAreNoSentMessages() {
         //given
         assertThat(sentMessages.count()).isZero();
-        operations.buildSubscription("returnEmptyListIfThereAreNoSentMessages", "topic", "subscription3", INVALID_ENDPOINT_URL);
+        Topic topic = operations.buildTopic("returnEmptyListIfThereAreNoSentMessages", "topic");
+        operations.createSubscription(topic, "subscription3", INVALID_ENDPOINT_URL);
 
         //when
-        Response response = management.subscription().getLatestUndeliveredMessages("returnEmptyListIfThereAreNoSentMessages.topic",
-                "subscription3");
+        Response response = management.subscription().getLatestUndeliveredMessages(topic.getQualifiedName(), "subscription3");
```

Comment:
I have just finished that review. Lots of changes but I do understand that they are necessary for migration process. I hope that in the future we will remove some parts of these functionality. Code looks good to me. 

:+1: